packages/side-nav/src/Search/Search.js

```diff
@@ -2,7 +2,7 @@ import React, { Component } from "react";
 import PropTypes from "prop-types";
 import { css, cx } from "emotion";
 import ThemeContext from "@hig/theme-context";
-import { Search24, Error24 } from "@hig/icons";
+import { Search24, CloseSUI } from "@hig/icons";
```

Comment:
Bryan mentioned here https://autodesk.slack.com/archives/C5FLF7R46/p1625863834122300?thread_ts=1625854174.121500&cid=C5FLF7R46 that it should be CloseLUI, I'm not sure what the difference is but we should find out.